pkgs/development/python-modules/simplesqlite/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, dataproperty
+, mbstrdecoder
+, pathvalidate
+, sqliteschema
+, setuptools
+, tabledata
+, typepy
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "SimpleSQLite";
+  version = "1.5.2";
+  format = "pyproject";
```

Comment:
```suggestion
  pyproject = true;
```

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/299429/commits/56bc22c06fbde480984b6817f99795daa2641759